### DISCUSSION TITLE
Relationship Rank Gift EXP calculations

### DISCUSSION THREAD

**Discussion by khoa301020** [lonqie/SchaleDB#41] [Ideas and Suggestions]
Since recently they have released the emblem (badge) system with bond level, I think a simulator to preview/calculate how many gifts we need to use to level up the bond to a specific level will be useful.

**Comment by lonqie** [comment]
Thanks for the suggestion, I will consider this feature in the future, however I think it is something that is perhaps better suited to a resource planning tool rather than Schale
